### PR TITLE
Temporary Access expiry alert on Applens

### PR DESCRIPTION
**Pull request by SharmaAjay19**
Users who are running on temporary access will receive a notification that informs them about how many days are remaining till their access expires. The notification also contains a direct link to apply for AppLens Access.
Home page:
image

Applens Dashboard:
image

View upon scrolling down in a detector, the banner still shows
image

They can hit close and ignore the notification, it will reappear the next time they open AppLens (if their access is still temporary).
When they hit close button:
image

### CODE REVIEW

**Comment by khaled-zayed** [comment]
❤

File: ApplensBackend/Controllers/TemporaryAccessController.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ public TemporaryAccessController(ICosmosDBHandler<TemporaryAccessUser> cosmosDBH
             temporaryAccessDurationInSeconds = temporaryAccessDays * 24 * 60 * 60;
         }
 
-        [HttpGet("requestAccess")]
+        [HttpPost("requestAccess")]
```
Comment (hforeste) [inline ApplensBackend/Controllers/TemporaryAccessController.cs]: Why did you need to make this a post?

Comment (SharmaAjay19) [reply ApplensBackend/Controllers/TemporaryAccessController.cs]: Because we are doing a write operation in backend for this. We write a document to cosmos db.
